app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class UsersController < AuthenticationController
+      def create
+        @user = User.create(user_params)
+        render json: '', status: :ok
```

Comment:
If the user can't be created it should return and error. Add the error cases please